pkg/volume/glusterfs/glusterfs.go

```diff
@@ -96,7 +96,6 @@ func (plugin *glusterfsPlugin) GetVolumeName(spec *volume.Spec) (string, error)
 	if err != nil {
 		return "", err
 	}
-
```

Comment:
First of all, these empty-line removals are not related to the change described in the commit message, and should not be sneaked in. *If* they are to be applied, they should be a patch of their own.

Secondly, I put several of those newlines into the text to increase readability. I think grouping blocks and lines of code by inserting newlines *greatly* increases the readability of code. I therefore request most of them to remain. If there is a "formatting policy I don't know of", I would be willing to disuss... :-)